tools/mtrace/mtrace-reader.py

```diff
@@ -35,4 +44,10 @@
     data_len = header[0]
     data = read_bytes[4:4+data_len]
 
-    os.write(sys.stdout.fileno(), data)
+    if (args.mark_chunks):
+        chunk_msg = "\n--- Chunk #{} start (size: {}) ---\n" .format(chunk_idx, data_len)
+        sys.stdout.write(chunk_msg)
+
+    sys.stdout.buffer.write(data)
+    sys.stdout.flush()
```

Comment:
@ujfalusi Paused for a moment on this. Not sure if this disables the buffering compared to old version, but I guess the flush is needed in practise here as the buffer.write() is used. Let's merge this version and return if this causes problems.